vcfapp/views.py

```diff
@@ -1,15 +1,19 @@
 from django.core.paginator import Paginator
 from django.http import HttpResponseRedirect, Http404
 from django.urls import reverse
-from pymongo import MongoClient
-from bson import ObjectId, regex
 from django.shortcuts import render, redirect
-from .forms import SingleVariantForm
+from bson import ObjectId, regex
+from pymongo import MongoClient
```

Comment:
Not needed in this script as used in utils.py now

Reply:
Done